src/pages/NotFound/NotFound.page.jsx

```diff
@@ -1,16 +1,48 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-
-import './NotFound.styles.css';
+import styled from 'styled-components';
 
 function NotFoundPage() {
+  const Wrapper = styled.section`
```

Comment:
I suggest creating a new file for the styled components to keep the styles and the component logic separated.